src/ListView/ListView.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+/**
+ * ListView Component for Patternfly React
+ */
+const ListView = ({ children, listViewClass }) => (
+  <div className={listViewClass}>
```

Comment:
maybe, I am not sure if classNames is useful for merging props, what I wrote equals
className={`list-pf${' ' + className}`}

with ClassNames I think you'd have to do (IIUC):
classNames({ 'list-pf': true, [`${className}`]: true });